package.json

```diff
@@ -66,6 +66,7 @@
     "canvas": "^2.10.1",
     "codemirror": "^6.0.0",
     "fast-xml-parser": "^4.0.10",
+    "imagemin-optipng": "^8.0.0",
```

Comment:
why are you adding this package if you never use it?

Reply:
If we are furthermore, is this a quality package we can trust? last release came out 3 years ago

Reply:
Good catch, I didn't mean to have this in the PR. This was part of devstack debugging.